browser/tor/test/onion_location_navigation_throttle_browsertest.cc

```diff
@@ -133,6 +133,11 @@ class OnionLocationNavigationThrottleBrowserTest : public InProcessBrowserTest {
 
     ui_test_utils::BrowserChangeObserver browser_creation_observer(
         nullptr, ui_test_utils::BrowserChangeObserver::ChangeType::kAdded);
+
+    content::TestNavigationObserver navigation_observer(
+        url, content::MessageLoopRunner::QuitMode::IMMEDIATE, false);
+    navigation_observer.StartWatchingNewWebContents();
+    navigation_observer.WatchExistingWebContents();
```

Comment:
why do we need to watch existing web_contents? Isn't `StartWatchingNewWebContents` sufficient for the to be created web_contents after button clicked?